packages/babel-plugin-transform-object-rest-spread/test/fixtures/object-spread/assignment/expected.js

```diff
@@ -1,5 +1,5 @@
 var _extends = Object.assign || function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; };
 
-z = _extends({ x }, y);
+z = _extends({}, { x }, y);
```

Comment:
It should not even be observably different even if there are getters/setters because according to spec the new properties should be using `defineProperty`.